### ISSUE TITLE
Policies Statement

### ISSUE THREAD

**Issue by runbeyond** [spectrumtrail/spectrum#49]
There is currently a policies link in the website footer that doesn't link to anything. Below is our Policies Statement.

RULES TO RUN BY:
 
1. We love dogs, cats, parakeets, and your pet goldfish just as much as you, but they won't be allowed to join you at this race. You are welcome to leave your pup on a leash with a person at the start/finish, though!

2. Pacers are only allowed on the second half of the double marathon. A pacer is only there for motivational support and safety, and thus cannot be used as a mule to carry any of your food/water/gear. 

3. You get your race swag (Goodwater bottle) for finishing, and only after finishing your race. You've got to earn it!

4. We can't give refunds or transfers. Once you purchase your race entry, we pay the land owners for your park pass, buy swag, pay the insurance company,  pay the timing company and photographer,  and on. That said, we're still very reasonable people and want to help you out, if you write a compelling request.

5. No cutting the course (ie CHEATING). Some days just aren't your day to race well. Realizing and respecting that bad race day by dropping out is WAY cooler than getting caught cheating. Oh, and if (when) we catch you, you will be asked to not race with us again. Ever.

6. We prefer you run without them, but if you're going to wear headphones, use only one earbud or turn it way down. You have to hear people trying to communicate with you. We have had a runner take off in the complete wrong direction just because she couldn't hear us screaming at her to stop. Her loss.
 
7. Most of the trails are single track or very narrow, so get used to saying "ON YOUR LEFT" when you pass. It also never hurts to tell that person something encouraging like, "NICE JOB". And while you're on a kind streak, a big "THANK YOU" to the volunteers is golden and earns you an extra beer.

8. If we have to postpone the race due to weather, we will push the race back to that afternoon or into the next day. We aren't afraid of a downpour, but we don't mess with lightning or hurricanes...or Sharknato.

**Comment by ColinW520** [comment]
@runbeyond -- I made the dogs text a little more robust to make sure they read the events page too.

 deployed, ready for testing & closure

**Comment by runbeyond** [comment]
Thanks, Colin! This works as expected. The red text at the top of the page encouraging users to read the terms on individual race pages is a great addition. Thank you.
